sensibility/miner/downloader.py

```diff
@@ -200,12 +202,15 @@ def fetch_repository(self, repo: RepositoryID) -> RepositoryMetadata:
             raise ValueError(f'Could not fetch info for {repo}')
         owner, name = RepositoryID.parse(info['nameWithOwner'])
         latest_commit = info['defaultBranchRef']['target']
+        licenseName = None
+        if info['licenseInfo'] is not None:
+            licenseName = info['licenseInfo']['name']
```

Comment:
I'd be really helpful if you added a link to the documentation: https://developer.github.com/v4/object/license/

Reply:
you mean adding as a comment above the line `206`?

Reply:
Yes, please!